docs/troubleshoot/troubleshooting.md

```diff
@@ -2,6 +2,29 @@
 
 To isolate and resolve Zowe&trade; problems, you can use the troubleshooting and support information.
 
+## How to start troubleshooting
+
+When you run into some issues and look for troubleshooting tips, the following steps may help you.
+
+1. Search the error message or error code that you have in log on [Zowe doc site](https://docs.zowe.org/). If there are existing solutions, then you could follow them.
```

Comment:
When you mention "log," I think we need to specify what kind of log. Is it an _error_ log? A _deploy_ log? A _personal_ log? I think "log" alone can be confusing. Also, I'd explain how to search. "Search the error message or code listed in your error log by using the Search bar in the Zowe Docs site" (Remember, [Zowe Docs](https://docs.zowe.org/) in capitalized: See the main page.)

Reply:
Also suggesting: "If there is an existing solution, follow instructions to troubleshoot."